torch/jit/frontend.py

```diff
@@ -178,6 +178,34 @@ def get_jit_class_def(cls, self_name):
     ctx = SourceContext(source, filename, file_lineno, leading_whitespace_len, False)
     return build_class_def(ctx, py_ast.body[0], methods, properties, self_name)
 
+def check_and_indent_multiline_strings(sourcelines):
+    """
+    This is a helper function which checks for multiline strings and
+    indents the strings by calculating the leading space and appending
+    the spaces to each line of the multiline string.
+
+    Arguments:
+        sourcelines: This is an array of source lines of the function
+    Returns:
+        This function returns the updated indented sources,i.e,sourcelines
+
+    """
+    indices = []
+    # Extract the start and end line number of the multiline string
+    for index, source in enumerate(sourcelines):
+        if '\"\"\"' in source and source.find('\"\"\"') == source.rfind('\"\"\"'):
```

Comment:
this is just a nit, but if you end up editing anyway for the above, one suggestion is to use a named constant to make the code more readable, e.g.
TRIPLE_QUOTE='\"....'
if TRIPLE_QUOTE in source and ...
maybe lower case looks less shouty .. feel free to ignore this or pick your own style
triple_quote='\" ... '